src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -270,8 +270,8 @@ private void requestEntityToBuildingIfRequired(Template.EntityInfo entityInfo)
                     {
```

Comment:
For the previous code, if the ItemFrame was empty, we just wouldn't request the ItemFrame. Now we will always request the ItemFrame, but only request the Item inside the frame, if it exists.